progress/tests.py

```diff
@@ -392,7 +394,10 @@ def test_progress_calc_on_vertical_with_detached_module(self):
         self.assertEqual(len(progress), 0)
 
     def test_receiver_on_course_deleted(self):
-        self._create_course(start=datetime(2010, 1, 1, tzinfo=UTC()), end=datetime(2020, 1, 1, tzinfo=UTC()))
+        self._create_course(
```

Comment:
We should add dual module_store support for all those tests where we are creating courses with `self._create_course`